source/SemVerAnalyzer/SemVerAnalyzer.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
+    <PackageReference Include="NuGet.Protocol" Version="6.9.1" />
```

Comment:
Since you are adding a brand new package and updating a whole bunch of logic, I believe it's worth bumping SemVerAnalyzer to a next major version - `4.0.0`.

Reply:
Ok, usually those things are done by the maintainer on release preparation in most workflows that I know. I will update the RELEASE-NOTES.md then as-well.